python/lsst/ip/isr/linearize.py

```diff
@@ -335,14 +336,13 @@ def toTable(self):
                           'FIT_PARAMS_ERR': self.fitParamsErr[ampName],
                           'RED_CHI_SQ': self.fitChiSq[ampName],
                           } for ampName in self.ampNames])
-
+        catalog.meta = self.getMetadata().toDict()
         tableList.append(catalog)
 
         if self.tableData:
             catalog = Table([{'LOOKUP_VALUES': value} for value in self.tableData])
             tableList.append(catalog)
-
-        return(catalog)
+        return(tableList)
```

Comment:
Was the previous returning of `catalog` just a bug?

Reply:
Just a bug.